docs/os/tutorials/add_shell.md

```diff
@@ -120,26 +116,46 @@ Once you have a connection set up, you can connect to your device as follows:
 To test and make sure that the Shell is running, first just hit <return>:
     
 ```no-highlight
-3534: >
+004543 compat>
 ```
 
-Remember, we turned the CONSOLE_PROMPT and the CONSOLE_TICKS on earlier. You can try some commands now:
+bletiny app uses legacy shell API to register command callbacks. That's why the prompt says "compat". All the legacy commands are registered to compat module to provide backward compatibility.
 
 ```no-highlight
-3609: > ?
-Commands:
-8841:     echo         ?    prompt     ticks     tasks  mempools
-8843:     date         b
-8844: > ticks off
- Console Ticks off
- > prompt off
- Prompt now off.
-ticks on
-33383: Console Ticks on
-
-33568:
-prompt on
-39108: Prompt now on.
-39108: >
+069822 compat> help
+070196 help
+070196 b
+070197 compat>
 ```
 
+You can run bletiny commands exactly the same way as in the old shell:
```

Comment:
Please remove this line, since a user working through this tutorial now will probably have not used the old shell, and just add a sentence to tell the user how to run a command.